chart/values.yaml

```diff
@@ -11,6 +11,9 @@ global:
   ##   - myRegistryKeySecretName
   ##
   imagePullSecrets: []
+  ## @param node.enableProvidedByTopology Enables workaround for upstream Kubernetes issue where nodes without the CSI plugin are still considered for scheduling.
```

Comment:
- Can you link the upstream kubernetes issue here?
- Can you explain that this setting should not be modified for existing clusters (and why)